features/FEATURE_BLE/ble/pal/GapEvents.h

```diff
@@ -306,7 +306,7 @@ struct GapAdvertisingReportEvent : public GapEvent {
     struct advertising_t {
         received_advertising_type_t type;
         connection_peer_address_type_t address_type;
-        const address_t& address;
+        address_t address;
```

Comment:
Any reason for that change ?

Reply:
The commit adding the initial change says "Fix advertising_t in GapEvents.h having a reference to an inexisting address_t instance". I'm not sure which usage of it caused the issue though.